app/models/user.rb

```diff
@@ -0,0 +1,36 @@
+require 'bcrypt'
+require 'dm-validations'
+
+
+class User
+  include DataMapper::Resource
+  include BCrypt
+
+
+  property :id,    Serial
+  property :name,  String
+  property :username, String, required: true, unique: true,
+  message:'Username invalid'
+  property :email,  String, required: true, unique: true,
+  message: "Email address invalid"
+  property :password , Text
+  attr_accessor :password_confirmation
+  validates_confirmation_of :password,
+  message: "Password does not match confirmation"
+  validates_format_of :email, as: :email_address
+
+
+  def password_new
+    @password_hashed ||= Password.new(password)
+  end
+
+  def password_new(new_password)
```

Comment:
Lint/DuplicateMethods: Method `User#password_new` is defined at both /app/app/models/user.rb:23 and /app/app/models/user.rb:27.